OpenRA.Mods.RA2/Orders/EnterGarrisonOrderTargeter.cs

```diff
@@ -0,0 +1,89 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2018 The OpenRA Developers (see AUTHORS)
```

Comment:
Wrong year.

Reply:
Do we really want all the years correct? I can go through and change all of the headers all at once. The more similar they are the better. If you want me to update this I can. I would just recommend not having these dissimilar from the current code. (this was copied from elsewhere in the project)

Reply:
Okay, we can leave this to @MustaphaTR's followup where he fixes these headers - as the codebase leader, the normalization should be his task anyway.